report_templates/index.Rmd.in

```diff
@@ -348,14 +348,14 @@ fetch_refgen_success <- ( fetched.refgenes != '')
 
 ## now we parse the gene features
 if( fetch_refgen_success ) {
-  print( paste( "In this run, the reference gene set was obtained with webfetch =", webfetch) )
+  cat( paste( "Reference gene set was obtained with webfetch =", webfetch) )
```

Comment:
These messages should no be printed as markdown, since they are rather status messages, than part of the report.